### PR TITLE
add pnoc nucleic acid extraction details

### PR DESCRIPTION
**Pull request by jharenza**

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -6,7 +6,14 @@ The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children
 Blood and tumor biospecimens from patients enrolled within the CBTTC were sent to the Children's Hospital of Philadelphia for RNA and/or DNA extraction.
 PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
 
-#### Nucleic acids extraction
+#### Nucleic acids extraction and library preparation
+
+##### PNOC samples
+Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using the Qiagen AllPrep Kit. 
+All RNAs used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNAs. 
+For library preparation, 500ng of nucleic acids were used as input for RNA-Seq and WXS.
```
Comment (cgreene) [inline content/03.methods.md]: Is WXS for whole-exome sequencing? Is this already defined elsewhere?

File: content/03.methods.md [hunk 2]
```diff
@@ -6,7 +6,14 @@ The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children
 Blood and tumor biospecimens from patients enrolled within the CBTTC were sent to the Children's Hospital of Philadelphia for RNA and/or DNA extraction.
 PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
 
-#### Nucleic acids extraction
+#### Nucleic acids extraction and library preparation
+
+##### PNOC samples
+Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using the Qiagen AllPrep Kit. 
+All RNAs used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNAs.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
All RNA used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNA.
```

File: content/03.methods.md [hunk 3]
```diff
@@ -6,7 +6,14 @@ The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children
 Blood and tumor biospecimens from patients enrolled within the CBTTC were sent to the Children's Hospital of Philadelphia for RNA and/or DNA extraction.
 PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
 
-#### Nucleic acids extraction
+#### Nucleic acids extraction and library preparation
+
+##### PNOC samples
+Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using the Qiagen AllPrep Kit.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using the Qiagen AllPrep Kit. 
```

File: content/03.methods.md (hunk 1 above)
Comment (jharenza) [reply content/03.methods.md]: > This is not terribly detailed. Would someone be able to follow these steps to end up with the same process? Did they specifically follow the Qiagen AllPrep Kit manufacturer's instructions? If so, we should say that and ideally link the instructions/protocol here.

I know. I have another email out to them asking for more specifics, but wanted to get in what I had so far. Mateusz will also be working on this section for CBTTC samples soon (today?), so wanted to get some of the details in so he can follow.

Comment (jharenza) [reply content/03.methods.md]: > Is WXS for whole-exome sequencing? Is this already defined elsewhere?

Yes, and I assumed we will be defining these abbreviations in the intro to describe the dataset, so did not do it here (unless you think each section will need this spelled out).

**Comment by cgreene** [comment]
@jharenza : can you put the exome sequencing one here: https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/33

Are you sure you want to use WXS instead of WES? I think WES is more common.

Once you put the abbreviation in there, I'm happy to approve this with the outstanding issues tracking the remaining needs for the section.

**Comment by jharenza** [comment]
> @jharenza : can you put the exome sequencing one here: #33
> 
> Are you sure you want to use WXS instead of WES? I think WES is more common.
> 
> Once you put the abbreviation in there, I'm happy to approve this with the outstanding issues tracking the remaining needs for the section.

Done - I prefer WES also, but WXS is the KF and CAVATICA convention....

**Comment by jaclyn-taroni** [comment]
WXS is what is used in the pbta-histologies.tsv file, correct? I can see a benefit to keeping it consistent between the data and the manuscript.

**Comment by jharenza** [comment]
> WXS is what is used in the pbta-histologies.tsv file, correct? I can see a benefit to keeping it consistent between the data and the manuscript.

correct

**Pull request merged**